specification/validated-relationships-service-api.yaml

```diff
@@ -481,36 +798,259 @@ components:
                           properties:
                             valueString:
                               type: string
-                              example: "LS1 4AP"
+                              example: "Child is too young"
 
-    QuestionnaireItem_RequestedServices:
+    QuestionnaireResponseItem_Patient:
       type: object
+      required:
+        - linkId
+        - text
+        - item
       properties:
         linkId:
           type: string
-          enum: ["requested_services"]
+          enum: ["patient"]
         text:
           type: string
-          example: "Requested services"
-        answer:
+          example: "patient"
+        item:
           type: array
           items:
-            anyOf:
+            oneOf:
               - type: object
+                required:
+                  - linkId
+                  - text
+                  - answer
                 properties:
-                  valueCoding:
-                    type: object
-                    properties:
-                      system:
-                        type: string
-                        enum: ['http://terminology.hl7.org/CodeSystem/consentaction']
-                      code:
-                        type: string
-                        enum: ['appointments','medicines','records','demographics']
-                      display:
-                        type: string
-                        example: "manage appointments"
-      
+                  linkId:
+                    type: string
+                    enum: ["patient_identifier"]
+                  text:
+                    type: string
+                    example: "NHS Number"
+                  answer:
+                    type: array
+                    minItems: 1
+                    maxItems: 1
+                    items:
+                      oneOf:
+                        - type: object
+                          properties:
+                            valueString:
+                              type: string
+                              example: "9000000002"
+
+              - type: object
+                required:
```

Comment:
`patient_name` only required when proxy applies. Will need to check this through logic rather than mandate in the schema. Child properties should be required though as per how you have them

Reply:
Removed the required links for name and DOB due to above logic depending on who is the applicant